02-Login-With-Profile/templates/dashboard.html

```diff
@@ -0,0 +1,57 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="assets/index.css" />
+    <link rel="stylesheet" href="assets/style.css" />
+    <title>Try Passage</title>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-B8V1R3HRNB"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || []; function
+      gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config',
+      'G-B8V1R3HRNB');
+    </script>
+  </head>
+  <body>
+
+    <div class="main-header">
+      <a href="https://passage.id/"><div class="passage-logo"></div></a>
+      <div class="header-text">Passage Go Example App</div>
+      <div class="spacer"></div>
+      <a href="https://passage.id/" class="link" style="color: white">Go to Passage</span></a>
+    </div>
+
+    <div class="main-container">
```

Comment:
for the profile dashboard, can you make a separate container that is wider so that profile looks better?

Reply:
Yep! On it now